### PR TITLE
change second factor in MFA to hardware-key and federated authenticat…

### PR DESCRIPTION
**Pull request by wijith7**
Changing the second factor of MFA to hardware-key and federated authentication as google

### CODE REVIEW

File: quick-start-guide/src/main/resources/bin/qsg.sh [hunk 1]
```diff
@@ -79,16 +79,15 @@ server_port=$4
...
-echo "|  So please make sure you have registered an application before  |"
-echo "|  continuing the script.                                         |"
+echo "|  Register OAuth 2.0 Application in Google if you did that .     |"
+echo "|  continue the script.                                           |"
 echo "|                                                                 |"
 echo "|  Do you want to continue?                                       |"
 echo "|                                                                 |"
 echo "|  Press y - YES                                                  |"
-echo "|  Press n - NO                                                   |"
+echo "|  Press n - N                                                    |"
```
Comment (ayshsandu) [inline quick-start-guide/src/main/resources/bin/qsg.sh]: Press n - NO?

File: quick-start-guide/src/main/resources/bin/qsg.sh [hunk 2]
```diff
@@ -79,16 +79,15 @@ server_port=$4
 echo
 echo "-------------------------------------------------------------------"
 echo "|                                                                 |"
-echo "|  To configure Twitter as a federated authenticator you          |"
-echo "|  have to register an application in https://apps.twitter.com/   |"
+echo "|  To configure google as a federated authenticator you should    |"
 echo "|                                                                 |"
-echo "|  So please make sure you have registered an application before  |"
-echo "|  continuing the script.                                         |"
+echo "|  Register OAuth 2.0 Application in Google if you did that .     |"
```
Comment (ayshsandu) [inline quick-start-guide/src/main/resources/bin/qsg.sh]: Refer a doc link

File: quick-start-guide/src/main/resources/bin/qsg.sh [hunk 3]
```diff
@@ -902,29 +901,30 @@ server_host=$3
...
+echo "-------------------------------------------------------------------------"
+echo "|                                                                       |"
+echo "|  To configure Hardware key as a second authentication factor,         |"
+echo "|  Please make sure you have put the                                    |"
+echo "|                                                                       |"
+echo "|  org.wso2.carbon.identity.sample.extension.authenticators-5.7.0.jar   |"
+echo "|  into the  <IS_HOME>/repository/components/dropins directory          |"
+echo "|                                                                       |"
+echo "|  and  sample-auth.war                                                 |"
+echo "|  into <IS_HOME>/repository/deployment/server/webapps folder           |"
```
Comment (ayshsandu) [inline quick-start-guide/src/main/resources/bin/qsg.sh]: Add step to restart

Comment (wijith7) [reply quick-start-guide/src/main/resources/bin/qsg.sh]: Done

**Pull request merged**